dobi.yaml

```diff
@@ -449,8 +460,6 @@ job=build-analyzer-deployer:
 job=run-analyzer-deployer-unit-tests:
   use: analyzer-deployer-build
   command: /bin/bash -c "source venv/bin/activate && cd analyzer-deployer && py.test -n auto -m 'not integration_test'"
-  depends:
-    - analyzer-deployer-build
```

Comment:
fyi, according to dobi docs, `use: $image` auto includes it in the depends: list

Reply:
👍 makes sense